observations/obs_converters/GSI2DART/gsi_to_dart.f90

```diff
@@ -53,7 +53,12 @@ program gsi_to_dart
 call mpi_initialize
 
 ! Print out some info
-call initialize_mpi_utilities('gsi_to_dart',communicator=my_communicator)
+call initialize_mpi_utilities('gsi_to_dart',communicator=mpi_comm_world)
+
+! The barrier makes sure the 'starting' banner from initialize_mpi_utilities is
+! printed before any subsequent print statements. Some tasks were printing the
+! error messages before the starting banner ... 
+call mpi_barrier(mpi_comm_world,ierr)
```

Comment:
this doesn't hurt but it's not a general fix for out-of-order error messages.  if this code fails later, it's going to do the same thing.  it doesn't hurt - but again i'd use my_communicator if you plan to leave this here.

Reply:
I understand that the messages from the tasks are always going to be out-of-order. I just wanted the 'starting program' banner to appear before any subsequent messages from the tasks.